spec.md

```diff
@@ -1704,6 +1712,10 @@ message ControllerServiceCapability {
       // This enables COs to, for example, fetch per volume
       // condition after a volume is provisioned.
       GET_VOLUME = 12 [(alpha_enum_value) = true];
+
+      // Indicates the SP supports the SINGLE_NODE_MULTI_WRITER access
+      // mode.
+      SINGLE_NODE_MULTI_WRITER = 13 [(alpha_enum_value) = true];
     }
```

Comment:
Why does this exists as a control-plane capability?

Reply:
COs require this capability in order to determine which access mode to supply for control plane operations that take `VolumeCapability`, like `CreateVolume()`. This capability will be used in the same way as the capability on the node side.

Reply:
@chrishenzie just to understand in bit more depth, but for COs with controller service operations, its just about knowing `SINGLE_NODE` or `MULTI_NODE` , isnt it ?  in that sense, `writer` thing ( single or multiple) is not relevant for CO's  control plane operations ?

Reply:
Yes, the number of writers here does not make a difference.